Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: help run stop test
+
+help: ## Print all commands (default)
+	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+install: ## Build the dockers
+	docker-compose build
```

Comment:
Are the composer dependencies  well installed?

Reply:
It should be because the docker specify this line: `RUN composer install -d /app; true`